speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
Nope it is not in my MOVIES slot, though (as we have discussed before) I am less convinced  the contents of the slots matter that much.

Maybe it would have worked if it was in the slot, but didn't need to be as it played fine when I said "play movie ghostbusters". Then when I added the articles to the sample utterances (still without ghostbusters in the slot) it then played fine when is said "play the movie ghostbusters" 

 You could test at your end with a movie you know is not in your MOVIES slot, or any movie even if it is not in your library. Just to see what Alexa hears.

Re the learning. I agree, giving feedback on those cards is very important in helping Alexa understand you better. I think that is the main way she learns, especially in UK/DE as we don't have the "voice training" feature that the US does.